projects/addon-editor/components/editor/editor.component.ts

```diff
@@ -135,6 +135,16 @@ export class TuiEditorComponent
         this.updateValue(value);
     }
 
+    onFocusChange(): void {
+        this.onActiveZone(true);
```

Comment:
Shouldn't `(tuiActiveZoneChange)` on `tuiWrapper` of editor component already take care of those focus/blur events?

Reply:
fixed